webapp/src/components/root/root.jsx

```diff
@@ -2,11 +2,32 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import {FormattedMessage} from 'react-intl';
 
-const Root = ({visible, close, theme}) => {
+import {menu} from '../../sub_menu';
+
+const Root = ({visible, close, theme, subMenuId}) => {
     if (!visible) {
         return null;
     }
 
+    let extraContent = '';
+    if (subMenuId) {
```

Comment:
I think you are keeping this content after close, so if you open the root modal from any other place, you are seeing the "submenu" extra content here.

By the way, probably the extra content should be more descriptive about himself. You can see there "Third item" without any context, probably you should have something like `Element clicked in the menu: Third item`

Reply:
Great catch, I solved it by dispatching an empty subMenuId on normal Root modal trigger.  Also added the additional descriptive text.